app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -241,17 +392,44 @@ export default function Navigation() {
             </div>
           </Hidden>
         </div>
-        <div className={classes.bug}>
-          <Hidden smDown>
-            <ExternalNavButton
-              route={handbookURL}
-              label={HELP}
-              labelVariant="h3"
-            />
-            <NavButton route={logoutRoute} label={LOG_OUT} />
-          </Hidden>
-          <ReportButton />
-        </div>
+
+        <Hidden smDown>
+          <div className={classes.menuContainer} ref={menuRef}>
+            <ReportButton />
+            <Button
+              className={classes.menuBtn}
+              onClick={() =>
+                setMenuOpen((prevMenuOpen: boolean) => !prevMenuOpen)
+              }
+            >
```

Comment:
Thanks for this, i made some research because i struggled finding is MUI's doc about aria-controls for menu but found a link from their docs that redirect to w3.org that made me understand the aria-controls https://www.w3.org/TR/wai-aria-practices/#menubutton ;)